src/modules/ordersTable/containers/OrdersTableWidget/hooks/useValidatePageUrlParams.ts

```diff
@@ -4,17 +4,17 @@ import { useLocation, useNavigate } from 'react-router-dom'
 
 import { Order } from 'legacy/state/orders/actions'
 
-import { LIMIT_ORDERS_PAGE_SIZE } from '../../../const/limitOrdersTabs'
-import { buildLimitOrdersUrl, parseLimitOrdersPageParams } from '../../../utils/buildLimitOrdersUrl'
+import { ORDERS_TABLE_PAGE_SIZE } from '../../../const/tabs'
+import { buildOrdersTableUrl, parseOrdersTableUrl } from '../../../utils/buildOrdersTableUrl'
```

Comment:
Absolute paths?

Reply:
Good question!
Asked in slack, would love to fix if we take a decision